### PR TITLE
Delete an app

### LINKED ISSUE #70: Delete an app

### PR DESCRIPTION
**Pull request by matm**
Fixes #70

@tbruyelle The returned `invalid character '<' looking for beginning of value` error is expected for now since the endpoint is not yet available.

### CODE REVIEW

File: cmd/applicationrm.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+	"github.com/keeneyetech/ecli/api"
+	"github.com/spf13/cobra"
+)
+
+// applicationrmCmd represents the applicationrm command
+var applicationrmCmd = &cobra.Command{
+	Use:   "rm APP_ID",
+	Short: "Delete an application",
+	Run: func(cmd *cobra.Command, args []string) {
```
Comment (tbruyelle) [inline cmd/applicationrm.go]: You can add between `Short` and `Run`  : 

```
Args: cobra.MinimumNArgs(1),
```

It asserts`args[0]` is present (no more need to check `len(args)`).

Comment (matm) [reply cmd/applicationrm.go]: nice, ty

Comment (matm) [reply cmd/applicationrm.go]: I'll use `ExactArgs(1)` instead.

Comment (matm) [reply cmd/applicationrm.go]: Fixed in 2aa8dcb

Comment (tbruyelle) [reply cmd/applicationrm.go]: Ah yes `ExactArgs` matches better our need, didn't know it exists !

**Pull request merged**